### PR TITLE
added 4th level A spells

### PR DESCRIPTION
**Pull request by bitclicker**
please let me know if any changes are needed, wanted to just do a few spells to make sure it matches the format you want before doing a whole bunch

### CODE REVIEW

File: _posts/spells/forth-level/2020-10-14-4-spells-aura-of-life.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Aura of Life
+permalink: "/spells/fourth-level/aura-of-life/"
+layout: spell-post
+categories: [Spells, Fourth Level]
+tags: [Paladin]
+source: "Basic Rules (PHB)"
+spell-level: "4th-level"
+school: Abjuration
+casting-time: "1 Action"
+range: "Self (30 ft)"
```
Comment (magicalmusings) [inline _posts/spells/forth-level/2020-10-14-4-spells-aura-of-life.md]: Self (30-foot radius)

File: _posts/spells/forth-level/2020-10-14-4-spells-aura-of-purity.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Aura of Purity
+permalink: "/spells/fourth-level/aura-of-purity/"
+layout: spell-post
+categories: [Spells, Fourth Level]
+tags: [Paladin]
+source: "Basic Rules (PHB)"
+spell-level: "4th-level"
+school: Abjuration
+casting-time: "1 Action"
+range: "Self (30 ft)"
```
Comment (magicalmusings) [inline _posts/spells/forth-level/2020-10-14-4-spells-aura-of-purity.md]: Self (30-foot radius)

File: _posts/spells/forth-level/2020-10-14-spells-4-arcane-eye.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+permalink: "/spells/fourth-level/arcane-eye/"
+layout: spell-post
+categories: [Spells, Fourth Level]
+tags: [Artificer, Wizard]
+source: "Basic Rules (PHB)"
+spell-level: "4th-level"
+school: Divination
+casting-time: "1 Action"
+range: "30 ft"
+components: "V, S, M *"
```
Comment (magicalmusings) [inline _posts/spells/forth-level/2020-10-14-spells-4-arcane-eye.md]: V, S, M (a bit of bat fur)

**Review by magicalmusings — CHANGES_REQUESTED** [review]
Thanks for doing an initial PR @bitclicker ! Glad you're taking contributions as seriously as you are. Feedback is below: 

- For Aura of Purity and Aura of Life, the filenames are backwards. They should be ```2020-10-10-spells-4-acid-splash.md``` as an example
- In the ```_posts/spells/``` directory, you misspelled fourth. Currently it's ```_posts/spells/forth-level/```
~~- I left individual comments about things in each file for fixes~~ (nvm, you hit those while I was reviewing. Way to be fast!) 

It should be good for approval and merging once you fix those issues. Thanks!

**Comment by bitclicker** [comment]
thanks @magicalmusings  sorry for those spelling mistakes i think i've got things the right way now

**Review by magicalmusings — APPROVED** [review]
Looks good for merge, thanks!

**Pull request merged**